modules/gitops-about-local-user-management-in-argo-cd.adoc

```diff
@@ -0,0 +1,19 @@
+// Module is included in the following assemblies:
+//
+// * accesscontrol_usermanagement/managing-local-users-in-argo-cd.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-about-local-user-management-in-argo-cd_{context}"]
+= About local user management in Argo CD
+
+Local users are intended for automation scenarios that require API tokens or for small teams where configuring single sign-on (SSO) is not necessary.
+
+The Argo CD Operator manages local users by:
+
+* Creating and managing user accounts defined in the Argo CD CR
+* Generating JSON Web Token (JWT) API tokens
+* Configuring token lifetimes and automatic renewal
+* Storing tokens securely in Kubernetes secrets
+* Cleaning up users and tokens when they are removed from the configuration
+
+For more information, see the link:https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/[Argo CD user management documentation].
```

Comment:
Should maybe clarify that the link provides info on local users and not info on the operator support for local users, maybe say something like `For more information on local users, see...`.

Reply:
Fixed.